spec/spec_helper.rb

```diff
@@ -1,5 +1,9 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
No need to do the coverage all the time. Only do it if `ENV['SIMPLECOV]` or something